llvm/lib/Passes/PassBuilderPipelines.cpp

```diff
@@ -2159,7 +2161,7 @@ ModulePassManager PassBuilder::buildO0DefaultPipeline(OptimizationLevel Level,
   CoroPM.addPass(GlobalDCEPass());
   MPM.addPass(CoroConditionalWrapper(std::move(CoroPM)));
 
-  invokeOptimizerLastEPCallbacks(MPM, Level);
+  invokeOptimizerLastEPCallbacks(MPM, Level, ThinOrFullLTOPhase::None);
```

Comment:
This doesn't look right. We should pass ThinOrFullLTOPhase through to buildO0DefaultPipeline.

Reply:
Good catch. I didn't notice the `bool LTOPreLink` argument.